libs/langchain/langchain/schema/document.py

```diff
@@ -17,6 +17,16 @@ class Document(Serializable):
         documents, etc.).
     """
 
+    @property
+    def lc_serializable(self) -> bool:
+        """Return whether or not the class is serializable."""
+        return True
+
+    @property
+    def lc_namespace(self) -> List[str]:
```

Comment:
This is not necessary, the only reason you find this in runnables is to maintain backwards compat with the folder they previously lived in